### ISSUE TITLE
Unable to shift-drag-deselect actors

### ISSUE THREAD

**Issue by matjaeck** [OpenRA/OpenRA#17322]
You can add actors to an existing selection by holding shift and dragging a selection box around non-selected actors. This however does not work in the opposite way for deselecting actors, since the selection is always combined when using a selection box - deselecting only works for single clicks.

**Comment by dragunoff** [comment]
I don't think this is a bug. As far as my experience goes that's how selection works in RTS games. Shift dragging is used only for adding to a selection. The reason for that most likely being that it would otherwise cause an inconvenience when dragging over a mix of selected and unselected actors (as you yourself have noted in #17323). And that would be a fairly common case because actors are usually moving around. 

Subtracting from a selection also IMO is a lot less useful than adding. However if we want to support it (I can imagine some cases where it can be useful) I'd suggest that this be attached to a different modifier or to a combination of modifiers.

We can look at how other types of software that has drag-selection handles that case:
- file managers only let you add to a selection when dragging (Ctrl-dragging in Windows explorer behaves like the Shift-dragging that you're describing here);
- graphics editing programs have a dedicated modifier for subtracting (`Alt` in Photoshop and GIMP);

**Comment by matjaeck** [comment]
Thanks @dragunoff, I agree with you that we should not change it like I proposed. I'll keep the issue open for discussion for now.

**Comment by BlackenedRA** [comment]
So I thought I was going crazy that shift deselect didn't work anymore. I'm not sure what changed, but deselection is pretty useful. I use it all the time when I have a large army and I want to move just a few units to a different area.

**Comment by abcdefg30** [comment]
@dragunoff fwiw I can use dragging to unselect things in my file manager.

I agree with @matjaeck and @BlackenedRA that deselecting is a very useful feature.
